src/test/scala/com/wix/mysql/EmbeddedMysqlTest.scala

```diff
@@ -143,6 +143,11 @@ class EmbeddedMysqlTest extends IntegrationTest {
         .withScripts(
           aMigrationWith("create table t1 (col1 INTEGER);"),
           aMigrationWith("create table t2 (col1 INTEGER);"))
+        .withCommands(
+          "create table t3 (col1 INTEGER);\n" +
+          "create table t4 (col2 INTEGER)",
+          "create table t5 (col3 INTEGER)"
```

Comment:
It's just a demonstration, that you may pass single expression and multiple expressions. Even with newlines.